src/components/panel/panel.spec.js

```diff
@@ -441,7 +442,21 @@ describe('$mdPanel', function() {
 
       openPanel(config);
       expect(PANEL_EL).toHaveClass(FULLSCREEN_CLASS);
+    });
+
+    it('should not show backdrop on open', function() {
```

Comment:
perhaps, `it('should default hasBackdrop to false', `

since you're testing the default behavior, not setting `hasBackdrop` to false.